sql/addons_view_v1.sql

```diff
@@ -0,0 +1,15 @@
+SELECT
```

Comment:
This should be named after the table it's generating, the current table is `addons_v2`, so probably `addons_v3.sql`.